src/driver/device.rs

```diff
@@ -98,6 +98,12 @@ impl Device {
             enabled_ext_names.push(ext::index_type_uint8::NAME.as_ptr());
         }
 
+        // Molten-vk doesn't support the full Vulkan feature set, hence the portability subset extension must be enabled.
+        #[cfg(all(target_os = "macos", feature = "macos-dynamic-molten-vk"))]
+        {
+            enabled_ext_names.push(khr::portability_subset::NAME.as_ptr());
+        }
+
```

Comment:
Shouldn't this be enabled for `target_os = "macos"` regardless of `macos-dynamic-molten-vk`? Same question below for `ash::khr::portability_enumeration`, etc...

Reply:
Examples from ash: 
https://github.com/ash-rs/ash/blob/55dc692946f52f0dff326bd13b1722d98f52fd2b/ash-examples/src/lib.rs#L317
https://github.com/ash-rs/ash/blob/59163296473aa6cb72ee0c4a63b25d7bb9823616/ash-examples/src/lib.rs#L233

(I have these in my screen-13 fork and have been meaning to make a PR)

Reply:
I'll leave a longer top-level comment with more details, but in short I've tried that, but it errors out at runtime when requesting those extensions, but not linking dynamically. 

I suspect (not verified though) this is because screen-13 defaults to using ash-molten, which in turn statically links against an outdated version of MoltenVK.